README.md

```diff
@@ -1,7 +1,16 @@
-# sis-tricks
+# ❄️🎈 sis-tricks 🎈❄️
 
-## [Multiple python files](https://github.com/blackary/sis-tricks/tree/main/multiple-python-files)
+<strong>A repo with the Streamlit Data Team's hacks for using Streamlit in Snowflake</strong>
```

Comment:
Just a nit -- my preference is not to add in html into markdown unless it's really something you can't do in markdown
```suggestion
**A repo with the Streamlit Data Team's hacks for using Streamlit in Snowflake**
```